lib/app/components/organisms/message_bar.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_form_builder/flutter_form_builder.dart';
+
+class MessageBar extends StatelessWidget {
+  const MessageBar({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      alignment: const Alignment(0, -1),
+      child: Container(
+        width: double.infinity,
+        height: 70,
+        decoration: BoxDecoration(
+          color: Colors.white,
+          border: Border.all(
+            color: Colors.black,
+          ),
+        ),
```

Comment:
```suggestion
      Alignment.bottomCenter,
      child: Container(
        padding: const EdgeInsets.only(
          left: 10,
          top: 10,
          right: 10,
          bottom: 40,
        ),
        width: double.infinity,
        height: 90,
        decoration: const BoxDecoration(
          color: Colors.white,
          border: Border(
            top: BorderSide(
              color: Colors.grey,
            ),
          ),
        ),
```
talk_room_view.dartで`Stack`の修正したので上記のように修正お願いします。

Reply:
メッセージバーの下の余白が余分に伸びていたのでbottomを40→10、heightを90→70にして下記の通り修正しました。ご確認お願いします。

```suggestion
Alignment.bottomCenter,
      child: Container(
        padding: const EdgeInsets.only(
          left: 10,
          top: 10,
          right: 10,
          bottom: 10,
        ),
        width: double.infinity,
        height: 70,
        decoration: const BoxDecoration(
          color: Colors.white,
          border: Border(
            top: BorderSide(
              color: Colors.grey,
            ),
          ),
        ),
```